contracts/seed/Seed.sol

```diff
@@ -157,8 +159,9 @@ contract Seed {
         seedToken = IERC20(_tokens[0]);
         fundingToken = IERC20(_tokens[1]);
         fee = _fee;
+        _seedTokenDecimal = uint8(_priceSeedTokenDecimal[1]);
 
-        seedAmountRequired = (hardCap * PRECISION) / _price;
+        seedAmountRequired = (hardCap * (10**_seedTokenDecimal)) / price;
```

Comment:
Do we really need to calculate seedAmountRequired in tokens and not in wei? Cannot it be done by frontend purely?

Reply:
And additionaly, it is really necessary to add decimals as a function parameter

Reply:
price would be in smallest unit of Funding token and that can either be in WEI if the token have decimal = 18, but for USDC with decimal 6, it would in 10^6. I would really like to have further discussion on this, just to make sure I'm not overcomplicating things here.